templates/.github/workflows/go-check.yml

```diff
@@ -18,6 +18,14 @@ jobs:
           go mod tidy
           diff go.mod go.mod.orig
           diff go.sum go.sum.orig
+      - name: go fmt
+        if: ${{ success() || failure() }} # run this step even if the previous one failed
+        run: |
+          out=$(gofmt -l .)
```

Comment:
consider adding -s; if you want to leave it out for now, add it as a TODO

Reply:
Makes sense, as long as we use it in the automation as well (which I'll make sure we do).